omero/sysadmins/unix/walkthrough/walkthrough_centos7.sh

```diff
@@ -15,11 +15,11 @@ yum -y install java-1.8.0-openjdk
 
 # install dependencies
 
-yum -y install python-{pip,devel,virtualenv,yaml,jinja2,tables}
+yum -y install python3-{pip,devel,virtualenv,yaml,jinja2,tables}
```

Comment:
```suggestion
yum -y install python3
```

Reply:
Those changes are made in omero-install so they will be overridden